src/Avalonia.Diagnostics/Diagnostics/Views/TreePageView.xaml.cs

```diff
@@ -39,7 +36,7 @@ protected void AddAdorner(object sender, PointerEventArgs e)
                 _adorner = new Rectangle
                 {
                     Fill = new SolidColorBrush(0x80a0c5e8),
-                    [AdornerLayer.AdornedElementProperty] = node.Visual
+                    [AdornerLayer.AdornedElementProperty] = node.Visual,
```

Comment:
Nit: Not needed comma.

Reply:
I actually prefer to leave trailing commas in initialization lists like this as it makes the diff cleaner when adding new entries.